samples/drivers/espi/sample.yaml

```diff
@@ -6,6 +6,7 @@ tests:
   sample.drivers.espi:
     tags: drivers espi
     filter: dt_compat_enabled("microchip,mec15xx-board-power")
+    platform_exclude: mec15xxevb_assy6853
```

Comment:
I think a harness might be better?

Reply:
@MaureenHelm I've updated README file and added proper harness with fixture setup.